bazel/cas/constants.go

```diff
@@ -13,6 +13,10 @@ const (
 	// Default buffer sizes
 	DefaultReadCapacity = 1024 * 1024
 
+	// Batch parallelism for underlying store operations
+	// NOTE if service implementation is changed, retest with setting <= 5
+	BatchParallelism = 5
```

Comment:
Why 5?

Reply:
Good call, that's an arbitrary low value that will hopefully see some speedup but not cause issues. Adding a comment about it.